interpreters/bas/bas_token.h

```diff
@@ -100,7 +100,7 @@ struct Identifier
 {
   struct Symbol *sym;
   enum ValueType defaultType;
-  char name[2/* ... */];
+  char name[0];
```

Comment:
This is C89 incompatible.

Reply:
no, do you have better method?

Reply:
@pkarashchenko ci is broken now, it's better to merge this patch ASAP, and provide a better fix later.

Reply:
Maybe encapsulate both in an `#ifdef __STDC` then c89 standard and `char name[2]` else `char name[0]`. I am not sure it will work, but if it does it should unblock the ci until we find a better long-term solution.

Reply:
@pkarashchenko please provide better method later, I will merge this patch to unblock ci.